src/ballet/toml/fd_toml.h

```diff
@@ -43,8 +44,12 @@ FD_PROTOTYPES_BEGIN
      inline table  | x={a=1,b=2} | subpod
      inline array  | x=[1,2]     | subpod (keys %d formatted cstrs)
      bool          | true        | int
-     integer       | -3          | ulong (two's complement)
-     float         | 3e-3        | double
+     dec int       | -3          | long
+     hex int       | 0x3         | ulong
```

Comment:
IMO these should map to `long` as well, according to the TOML spec there is no ulong, and the type of a field shouldn't change just because the user changed it from 0x1 to 1

Reply:
Sounds good